tools/rhythm-ui/README.md

```diff
@@ -0,0 +1,48 @@
+# Rhythm UI
+
+## Getting Started
+
+### Build
+To build the latest bundle of rhythm UI web components - run `yarn build`. 
+
+This will build the library into the `/dist` directory.
+
+For modern browsers that enable ES modules, use `/dist/module/rhythm-ui.<version>.js`.
+
+For older browsers that do not support modules, you can use the system js version of the build `/dist/nomodule/rhythm-ui.<version>-sys.js`
+
+
+### Include
+Once you have built the library, you can include the js file on your page as per usual, e.g.
+```html
+<script type="module">
+    import './dist/module/rhythm-ui.1.0.0.js';
+</script>
+```
+or 
+```html
+<script type="text/javascript" src="./dist/module/rhythm-ui.1.0.0.js"></script>
+```
+
+### Browser Support
+Please note that support for web components is still limited and will require polyfills.
```

Comment:
will require polyfills **for older browsers**

Reply:
@jennasalau fixed :)